tests/fixtures/additionalInformation.json

```diff
@@ -0,0 +1,303 @@
+            "format": "uri",
+            "minLength": 1,
+            "description": "URL where the tool can be found/downloaded."
+          },
+          "name": {
+            "type": "string",
+            "minLength": 1,
+            "description": "The name of the tool."
+          }
+        }
+      },
+      "access_point": {
+        "type": "object",
+        "required": [
+          "type",
+          "url"
+        ],
+        "properties": {
+          "url": {
+            "type": "string",
+            "format": "uri",
+            "minLength": 1,
+            "description": "The URL of the web service."
+          },
+          "type": {
+            "enum": [
+              "REST",
+              "SOAP",
+              "SPARQL",
+              "Other"
+            ],
+            "type": "string",
+            "default": "REST",
+            "description": "The type of web service being run. Three most commonly-used values are supplied (REST, SOAP, SPARQL), with “Other” also provided to allow for any other types / unknown service types."
+          },
+          "example_url": {
+            "type": [
+              "string",
+              "null"
+            ],
+            "format": "uri",
+            "description": "An example query for the service."
+          },
+          "documentation_url": {
+            "type": [
+              "string",
+              "null"
+            ],
+            "format": "uri",
+            "description": "The URL where documentation related to the web service can be found."
+          }
+        }
+      },
+      "data_process": {
+        "type": "object",
+        "required": [
+          "name",
+          "type"
+        ],
+        "properties": {
+          "url": {
+            "type": "string",
+            "format": "uri",
+            "minLength": 1,
+            "description": "The URL at which the named data process can be found."
+          },
+          "name": {
+            "type": "string",
+            "minLength": 1,
+            "description": "The name of the process being performed."
+          },
+          "type": {
+            "enum": [
+              "data access",
+              "data curation",
+              "data release",
+              "data versioning"
+            ],
+            "type": "string",
+            "default": "data access",
+            "description": "Data access is used when the process is accessing data, e.g. searching, browsing, viewing, and/or visualizing the data. Data curation is used when the process provides data curation tasks, e.g. submitting and/or editing the data. Data release is used when the process is releasing data, e.g. downloading and/or exporting the data. Data versioning is used when the process is versioning data, e.g. upload of new versions of data."
+          }
+        }
+      }
+    },
+    "description": "Properties specific to repositories",
+    "additionalProperties": false
+  },
+  "data": {
```

Comment:
Is this something left over from debugging, or is it used somewhere?

Reply:
This is a test fixture im using to run the schemas tests. What's the problem?

Reply:
None. I couldn't spot where it was used and wondered about it.